person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permision: true)
+    @id = Random.rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permision = parent_permision
+  end
+
+  attr_accessor :age, :name
+  attr_reader :id
```

Comment:
- [ ] **Optional:** It is conventional to have the attribute at the top pf the class before the initialize method.